model/riscv_insts_zba.sail

```diff
@@ -0,0 +1,88 @@
+/* ****************************************************************** */
+/* This file specifies the instructions in the BitManip Zba extension */
+
+
+/* ****************************************************************** */
+union clause ast = RISCV_SLLIUW : (bits(6), regidx, regidx)
+
+mapping clause encdec = RISCV_SLLIUW(shamt, rs1, rd) if haveZba() & sizeof(xlen) == 64
```

Comment:
Guards need to be on both directions of the mapping (repeated throughout)

Reply:
Done.